examples/examples.yaml

```diff
@@ -21,3 +21,35 @@ examples:
   - id: "tensorflow-akka"
     path: "tensorflow-akka"
     description: This app shows how to serve a Tensorflow model using Akka
+  - id: "sensor-data-scala-snippets"
+    path: "snippets/modules/ROOT/examples/sensor-data-scala"
+    description: |
+      Copy of sensor-data-scala to be able to include the files in the documentation
+  - id: "akkastreams-scala-snippets"
+    path: "snippets/modules/ROOT/examples/akkastreams-scala"
+    description: |
+      Set of files containing the Scala Akka streams code snippets used in the documentation
+  - id: "akkastreams-java-snippets"
+    path: "snippets/modules/ROOT/examples/akkastreams-java"
+    description: |
+      Set of files containing the Java Akka streams code snippets used in the documentation
+  - id: "spark-scala-snippets"
+    path: "snippets/modules/ROOT/examples/spark-scala"
+    description: |
+      Set of files containing the Scala Spark code snippets used in the documentation
+  - id: "build-akka-streamlets-java-snippets"
+    path: "snippets/modules/ROOT/examples/build-akka-streamlets-java"
+    description: |
+      Set of files containing the Scala code snippets used in the build-akka-streamlets documentation
```

Comment:
Is there a good reason to keep this separate from `akkastreams-java` (and the same for the scala side)? Whot not have less top-level containers and (if/when necessary) just use folders to structure the contents of such projects.

I had expected one single snippets project or perhaps, if that made more sense, one per runtime. I'm not sure this additional directory structure serves a good purpose.

Reply:
Every extra sbt project we add here requires us to updates its dependencies, etc. I'd prefer the smallest possible number of snippet projects and then use internal directories to add structure where needed. If the Java vs Scala difference is tricky then that could be a necessary reason for having more than 1 snippets (e.g. 2) projects.

Reply:
One of the reasons to have different project is that most of them define a blueprint
And I remember that we had problems with sbt projects defining multiple Cloudflow app subproject, and we include blueprint in the documentation, which have to be verified.
So I went the easy way, one project per blueprint related bloc of snippets.

Also, having one, or a couple of, big sbt project will only be moving the complexity.
The `build-*-streamlets-*` projects already contain each 4 library projects, one for each steps of the explanation, and one app project for the blueprint.

But I should be able to group some of the simple projects.

Reply:
ok, thanks. Just trying to keep a handle on the complexity 😉